### PR TITLE
Applied fixes from FlintCI

### PR DESCRIPTION
**Pull request by soullivaneuh**

### CODE REVIEW

File: src/EventListener/AssetsInstallCommandListener.php [hunk 1]
```diff
@@ -75,7 +75,7 @@ public function copySonataCoreBundleAssets(ConsoleTerminateEvent $event)
         $this->execute($event->getInput(), $event->getOutput(), $application);
     }
 
-    protected function execute(InputInterface $input, OutputInterface $output, FrameworkApplication $application): int
+    private function execute(InputInterface $input, OutputInterface $output, FrameworkApplication $application): int
```
Comment (VincentLanglet) [inline src/EventListener/AssetsInstallCommandListener.php:78]: This seems weird

File: src/Command/CreateClassCacheCommand.php [hunk 2]
```diff
@@ -70,7 +70,7 @@ public function execute(InputInterface $input, OutputInterface $output)
 
         $output->write('<info>Writing cache file ...</info>');
         ClassCollectionLoader::load(
-            include($classmap),
+            include ($classmap),
```
Comment (VincentLanglet) [inline src/Command/CreateClassCacheCommand.php:73]: This conflict with phpcs fixer

**Review by VincentLanglet — CHANGES_REQUESTED** [review]
@OskarStark Do you know what's happening ?

File: src/EventListener/AssetsInstallCommandListener.php [hunk 3]
```diff
@@ -37,7 +37,7 @@ final class AssetsInstallCommandListener
     public const METHOD_ABSOLUTE_SYMLINK = 'absolute symlink';
     public const METHOD_RELATIVE_SYMLINK = 'relative symlink';
 
-    protected static $defaultName = 'assets:install';
+    private static $defaultName = 'assets:install';
```
Comment (OskarStark) [inline src/EventListener/AssetsInstallCommandListener.php:40]: this seems weird too

File: src/Command/CreateClassCacheCommand.php (hunk 2 above)
Comment (OskarStark) [reply src/Command/CreateClassCacheCommand.php:73]: I created a new php-cs-fixer 2.17.0 release in the morning

File: src/EventListener/AssetsInstallCommandListener.php (hunk 1 above)
Comment (OskarStark) [reply src/EventListener/AssetsInstallCommandListener.php:78]: can you report an issue on php-cs-fixer please?

**Pull request closed without merging**